nonprofit-board-management.php

```diff
@@ -71,7 +71,7 @@ public function __construct(){
           add_action('wp_dashboard_setup', array( $this, 'add_board_members_dashboard_widget' ) );
 
           //Remove the help tabs for all board members
-          add_filter( 'contextual_help', array( $this, 'remove_help_tabs' ), 10, 3 );
+          add_action( 'admin_head', array( $this, 'remove_help_tabs' ) );
```

Comment:
Do you think using the `in_admin_header` filter might be better since it's called just before `$current_screen->render_screen_meta()` in wp-admin\admin-header.php? My fear is that help tabs will be added after we have cleared them out. That said, go with what you think is best since you spent more time looking at this than I have.